docs/theme.config.tsx

```diff
@@ -124,13 +124,13 @@ const config: DocsThemeConfig = {
         />
         <meta name="msapplication-TileColor" content="#000000" />
         <meta name="theme-color" content="#000" />
-        <meta name="twitter:card" content="summary_large_image" />
-        <meta name="twitter:site" content="@turborepo" />
-        <meta name="twitter:creator" content="@turborepo" />
+        <meta name="x:card" content="summary_large_image" />
+        <meta name="x:site" content="@turborepo" />
+        <meta name="x:creator" content="@turborepo" />
```

Comment:
does this work? docs don't seem to mention these tags? https://developer.twitter.com/en/docs/twitter-for-websites/cards/guides/getting-started

Reply:
seems not, pushed a fix